modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpNextgenClientCodegen.java

```diff
@@ -149,7 +149,7 @@ private ModelsMap postProcessModelsMap(ModelsMap objs) {
                     propType = prop.dataType;
                 }
 
-                if ((!prop.required || prop.isNullable)) { // optional or nullable
+                if ((!prop.required || prop.isNullable) && propType != "mixed") { // optional or nullable but not mixed
```

Comment:
FYI. Filed https://github.com/OpenAPITools/openapi-generator/pull/17116 to compare strings using `.equals` instead of `!=`